readthedocs/vcs_support/backends/git.py

```diff
@@ -56,12 +56,9 @@ def repo_exists(self):
         return code == 0
 
     def fetch(self):
-        code, _, err = self.run('git', 'fetch', '--tags', '--prune')
+        code, _, _ = self.run('git', 'fetch', '--tags', '--prune')
         if code != 0:
-            raise ProjectImportError(
-                "Failed to get code from '%s' (git fetch): %s\n\nStderr:\n\n%s\n\n" % (
-                    self.repo_url, code, err)
-            )
+            raise RepositoryError
```

Comment:
Do we want to lose the specifics of these errors?

Reply:
I think surfacing an error that is helpful provides more user guidance -- that is, showing a failed checkout command isn't clear to the user that the problem is their configuration. I think the ultimate fix here is to do both and show the command execution as a `BuildCommand`. I removed this for now, for consistency on errors and because displaying the stderr output like this looked sloppy. I think we can pretty quickly move towards combining execution environments though.

Reply:
I think that having both in the future is better.

I like the idea of communicate the "common problem" to the user in a very simplified way, but also show the output (that could be used when debugging something and could help the user to understand it better -also if the problem is something different that we are reporting as simplified way)